library/std/src/sys/mod.rs

```diff
@@ -80,12 +80,27 @@ cfg_if::cfg_if! {
     }
 }
 
+// todo: does android have its own version of these functions?
```

Comment:
@devsnek / @workingjubilee it looks like you authored the above log wrapper - do `android::{log2f16, log2f128}` exist, or what should be done here?

Reply:
this is an excellent question. i don't know the answer. i would guess that these should be implemented for f16 and f128, but that is just a guess: https://github.com/rust-lang/rust/blob/master/library/std/src/sys/unix/android.rs#L48-L56

Reply:
Awesome, thank you for the quick response and the link. I have gone ahead and added similar implementations